### ISSUE TITLE
Clarify docs around Action.title

### ISSUE THREAD

**Issue by zombiezen** [sandstorm-io/sandstorm#1912]
package.capnp docs say that Action.title is obsolete in favor of Action.nounPhrase. However, `spk init` doesn't add nounPhrase (only title), and further, Sandstorm will log a stack trace if you try to upload a package that contains actions without title.

```
Exception while invoking method '/userActions/insert' Error: Match error: Missing key 'title'
    at check (packages/check.js:138:15)
    at insert (server/install-server.js:28:5)
    at packages/allow-deny/allow-deny.js:229:13
    at Array.every (native)
    at Function._.every._.all (packages/underscore.js:253:62)
    at [object Object].CollectionPrototype._validatedInsert (packages/allow-deny/allow-deny.js:228:9)
    at [object Object].m.(anonymous function) (packages/allow-deny/allow-deny.js:156:46)
    at maybeAuditArgumentChecks (packages/ddp-server/livedata_server.js:1704:12)
    at packages/ddp-server/livedata_server.js:711:19
    at [object Object]._.extend.withValue (packages/meteor.js:1100:17)
Sanitized and reported to the client as: Match failed [400]
```

**Comment by paulproteus** [comment]
Hey @zarvox,

Are you OK with trying to handle this sometime in the next week or two? As I recall, you introduced nounPhrase. I am OK owning it if you prefer, but it'd take me a little bit of source code reading to make sure I understand how it's supposed to work.

**Comment by zombiezen** [comment]
I should add that along with the stack trace, the app disappears from the list of apps.  It would be nice to have package checks happen before installing the package.

**Comment by zarvox** [comment]
Yeah, I'll take a look at this on this Friday or next.

**Comment by paulproteus** [comment]
Thanks! :D

**Comment by zarvox** [comment]
Closing, fixed in #2178
